### PR TITLE
#10568 Add Org filter for intratrade and intracommunity reports 

### PR DESCRIPTION
**Pull request by cristinamghita**
#10568 Add Org filter for intratrade and intracommunity reports

### CODE REVIEW

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/reports/intratrade/report.jrxml [hunk 1]
```diff
@@ -166,11 +175,11 @@ order by row_number;]]>
 				<textFieldExpression><![CDATA[$R{deliveredfromcountry}]]></textFieldExpression>
 			</textField>
 			<textField isStretchWithOverflow="true" isBlankWhenNull="true">
-				<reportElement x="50" y="45" width="523" height="14" uuid="285aba6a-0a15-4208-af82-b6562c35578b"/>
+				<reportElement x="50" y="56" width="523" height="14" uuid="285aba6a-0a15-4208-af82-b6562c35578b"/>
 				<textElement>
 					<font fontName="Arial" size="10" isBold="true"/>
 				</textElement>
-				<textFieldExpression><![CDATA[$R{Month}+"  " + $F{period}]]></textFieldExpression>
+				<textFieldExpression><![CDATA[$F{orgname}]]></textFieldExpression>
```
Comment (metas-ts) [inline backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/reports/intratrade/report.jrxml:182]: don'T we want month & period anymore

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/75-de.metas.fresh.reports/5577960_sys_gh10568_AddOrgFilter.sql [hunk 2]
```diff
@@ -0,0 +1,42 @@
+-- 2021-02-01T09:39:40.879Z
+-- URL zum Konzept
+INSERT INTO AD_Process_Para (AD_Client_ID,AD_Element_ID,AD_Org_ID,AD_Process_ID,AD_Process_Para_ID,AD_Reference_ID,ColumnName,Created,CreatedBy,DefaultValue,Description,EntityType,FieldLength,Help,IsActive,IsAutocomplete,IsCentrallyMaintained,IsEncrypted,IsMandatory,IsRange,Name,SeqNo,Updated,UpdatedBy) VALUES (0,113,0,584656,541919,19,'AD_Org_ID',TO_TIMESTAMP('2021-02-01 10:39:35','YYYY-MM-DD HH24:MI:SS'),100,'1000000','Organisatorische Einheit des Mandanten','U',0,'Eine Organisation ist ein Bereich ihres Mandanten - z.B. Laden oder Abteilung. Sie kĂ¶nnen Daten ĂĽber Organisationen hinweg gemeinsam verwenden.','Y','N','Y','N','Y','N','Sektion',5,TO_TIMESTAMP('2021-02-01 10:39:35','YYYY-MM-DD HH24:MI:SS'),100)
```
Comment (metas-ts) [inline backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/75-de.metas.fresh.reports/5577960_sys_gh10568_AddOrgFilter.sql:3]: are the new parameters mandatory? if not, i think they should be..

**Review by metas-ts — COMMENTED** [review]
please check my remarks. might or might not be problems..

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/75-de.metas.fresh.reports/5577960_sys_gh10568_AddOrgFilter.sql (hunk 2 above)
Comment (cristinamghita) [reply backend/de.metas.fresh/de.metas.fresh.base/src/main/sql/postgresql/system/75-de.metas.fresh.reports/5577960_sys_gh10568_AddOrgFilter.sql:3]: yes, they are

File: backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/reports/intratrade/report.jrxml (hunk 1 above)
Comment (cristinamghita) [reply backend/de.metas.fresh/de.metas.fresh.base/src/main/jasperreports/de/metas/reports/intratrade/report.jrxml:182]: Yes, we want; is just moved lower,but is there.

**Review by metas-ts — APPROVED** [review]

**Pull request merged**